aibolit/patterns/multiple_while/multiple_while.py

```diff
@@ -1,5 +1,4 @@
-import javalang
-
+from aibolit.utils.ast import AST, ASTNodeType
```

Comment:
This strange diff from another PR looks really weird. Can we update branches somehow to get rid of it?